cpmpy/expressions/variables.py

```diff
@@ -468,13 +468,279 @@ def sum(self, axis=None, out=None):
         """
             overwrite np.sum(NDVarArray) as people might use it
 
-            does not actually support axis/out... todo?
+            does not actually support out... todo? I think we should not support out!
```

Comment:
I don't know what it does, if it does not make sense for us then fine to raise against it, no further comment needed